subschedule.go

```diff
@@ -339,6 +341,8 @@ type SubscriptionSchedulePhase struct {
 	InvoiceSettings *SubscriptionScheduleInvoiceSettings `json:"invoice_settings"`
 	// Subscription items to configure the subscription to during this phase of the subscription schedule.
 	Items []*SubscriptionSchedulePhaseItem `json:"items"`
+	// Set of [key-value pairs](https://stripe.com/docs/api/metadata) that you can attach to a phase. Metadata on a schedule's phase will update the underlying subscription's `metadata` when the phase is entered. Updating the underlying subscription's `metadata` directly will not affect the current phase's `metadata`.
+	Metadata map[string]string `json:"metadata"`
```

Comment:
When can we expect the Metadata field to be added to the `SubscriptionSchedulePhaseItem` definition? It's listed as a preview feature in the API docs, but unsure what that means for the codegen. https://stripe.com/docs/api/subscription_schedules/object#subscription_schedule_object-phases-items-metadata

Reply:
Hi @OilyLime, if it's listed as a preview feature in the API Reference that means it is not officially in the Public API and will not be included in the SDKs until it is officially released. The reason is that preview features are subject to change or removal and we limit the public SDKs to stable features to avoid breaking changes.